src/applications/vaos/services/mocks/index.js

```diff
@@ -184,14 +186,16 @@ const responses = {
   },
   'PUT /vaos/v0/preferences': { data: { attributes: {} } },
   'POST /vaos/v2/appointments': (req, res) => {
-    return res.json({
-      data: {
-        id: '32152',
-        attributes: {
-          ...req.body,
-        },
+    const submittedAppt = {
+      id: `mock${currentMockId}`,
+      attributes: {
+        ...req.body,
+        start: req.body.slot ? req.body.slot.start : null,
```

Comment:
For direct scheduled appointments, we need to set an actual start date, since we'll get one when using the actual services.